jwst/regtest/test_miri_mrs.py

```diff
@@ -1,14 +1,36 @@
 """Regression tests for MIRI MRS modes"""
+from pathlib import Path
 import pytest
 
+from jwst.associations import load_asn
+from jwst.lib.suffix import replace_suffix
+
 from . import regtestdata as rt
 
 
+# Define artifactory source and truth
+BIGDATA_PATH = 'miri/mrs'
```

Comment:
Could rename this path to something other than BIGDATA?  I fear confusion with TEST_BIGDATA.